src/AS_02_IAB.cpp

```diff
@@ -272,7 +272,7 @@ AS_02::IAB::MXFWriter::Finalize() {
 
     byte_t clip_buffer[8] = { 0 };
```

Comment:
The section between /* write clip length */ and /* write footer */ looks like FinalizeClip() function. Is there any reason you are not using it?

Reply:
Do you mean https://github.com/cinecert/asdcplib/blob/58628942da20e483f5c3076d6c14ff4bef71fc71/src/h__02_Writer.cpp#L356 ?

If so, that function assumes a fixed frame size if I am not mistaken.

Reply:
Ah, yes, correct. Nevermind.